pkg/charset/charset.go

```diff
@@ -40,6 +40,7 @@ const (
 // htmlEncodingNames represents a mapping of DICOM charset name to golang encoding/htmlindex name.  "" means
 // 7bit ascii.
```

Comment:
can you update this comment please as I don't think it applies anymore?

Reply:
Hi @suyashkumar ,
Thank you for merging this.

I considered correcting the comment, but I thought this comment didn't say anything wrong, so I left it alone.

In DICOM, "" represents 7-bit ascii (equivalent to ISO_IR 6, which is an abbreviation that I won't go into detail about).
So, I think the following comment is correct.
> "" means 7bit ascii.

On the other hand, when mapping 7bit ascii (ISO_IR 6) to a golang name, it seems to use iso-8859-1.
So I think the following comment is also correct.

> htmlEncodingNames represents a mapping of DICOM charset name to golang encoding/htmlindex name.

Is there something wrong with my understanding? (I don't know much about it, so I might be wrong...)

Regards